src/main/java/com/purbon/kafka/topology/TopologyBuilderConfig.java

```diff
@@ -58,6 +58,7 @@
   static final String PROJECT_PREFIX_FORMAT_CONFIG = "topology.project.prefix.format";
   static final String TOPIC_PREFIX_SEPARATOR_CONFIG = "topology.topic.prefix.separator";
   static final String TOPOLOGY_VALIDATIONS_CONFIG = "topology.validations";
+  static final String CONNECTOR_ACL_TOPIC_CREATE = "topology.connector.acl.topic.create";
```

Comment:
Yes, I agree if the same can be implemented for RBAC. I thought about it...but was not sure how this can be supported for RBAC. So you can implement the same feature for RBAC?

Reply:
yes. Please, for now, name it method agnostic. thanks a lot!